### ISSUE TITLE
blocklist / allowlist for tags

### ISSUE THREAD

**Issue by seigneurfuo** [ahodesuka/ahoviewer#123]
Hello. 
Would it be possible to implement a tag blocklist / allowlist system ?
Maybe this is already planned for a future version?
Thanks.

**Comment by ahodesuka** [comment]
Yeah, I can add a tag blacklist feature, should be pretty simple.
